odoo/addons/base/models/ir_qweb.py

```diff
@@ -618,9 +619,11 @@ def _get_view_id(self, template):
 
     @QwebTracker.wrap_compile
     def _compile(self, template):
+        ref = None
         if isinstance(template, etree._Element):
             self = self.with_context(is_t_cache_disabled=True)
-            ref = None
+        elif isinstance(template, str) and '/' in template and not any(ch in template for ch in ('<', '>')):
+            template = etree.fromstring(file_open(template, 'rb').read())
```

Comment:
You need to close the `file_open`

```suggestion
            with file_open(template, 'rb') as fp:
                template = etree.fromstring(fp.read())
```